x-pack/test/fleet_api_integration/apis/epm/install_remove_assets.ts

```diff
@@ -399,6 +399,11 @@ const expectAssetsInstalled = ({
       id: 'sample_ml_module',
     });
     expect(resMlModule.id).equal('sample_ml_module');
+    const resSecurityRule = await kibanaServer.savedObjects.get({
+      type: 'security-rule',
+      id: 'sample_security_rule',
+    });
+    expect(resSecurityRule.id).equal('sample_security_rule');
```

Comment:
@nchaulet this test is still failing on me. i realized there wasn't a saved object definition for `security-rule` so I added one to the security solution plugin and register it within there, but I'm still getting failures here.

Reply:
or @ruflin, any ideas?
and I commented on the wrong line. It's the line above that returns a 404

Reply:
I'll ping here @jen-huang @nchaulet @skh @jfsiii for help, they will know best.

Reply:
Does this work outside of the test case, i.e. can you install a package with a `security-rule` and retrieve it with the saved objects HTTP API?

Reply:
Ended up hunting this down to a mapping issue:
`mapping set to strict, dynamic introduction of [author] within [security-rule] is not allowed`